app/pulp/app/db-reset.sh

```diff
@@ -1,9 +1,15 @@
-#!/bin/bash -x
-
-# This assumes you're running in vagrant.
-echo "drop database pulp; create database pulp owner pulp" | sudo -u postgres psql
-
-rm -rf migrations
```

Comment:
Can you explain the motivation for this? When I reset the database in my own work, I had no reason to keep the migrations.